webui/framework/src/main/java/org/exoplatform/webui/exception/CSRFException.java

```diff
@@ -1,6 +1,6 @@
 package org.exoplatform.webui.exception;
 
-public class CSRFException extends Exception{
+public class CSRFException extends RuntimeException{
```

Comment:
Why did you change the exception type ? Is there a particular reason ?

Reply:
There is no major difference, but RuntimeException is an unchecked exception, and those exception are used when there is no recover for the thrown issue which is dealing with our situation where we cant' do anything except reloading the page when this exception comes to happen

Reply:
After thinking, I restore the Exception as we are handling it in a try catch clause